tensorflow_addons/image/median_filter_2D_test.py

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
Please add this test and the above code to the BUILD file so that it can be packaged. Examples should be in the [BUILD file](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/image/BUILD) though we should have better documentation so feel free to ask if it's not clear.

Reply:
For clarity the src code should be added here:
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/image/BUILD#L7

And a py_test should be added to run the test

Reply:
added